apps/algorithm/search/README.md

```diff
@@ -14,27 +15,32 @@
 ```
 
 Failed build output contains this:
+
 ```
 	martletplace_pgsync exited with code 1
 ```
 
-If your output does not match the successful output or contains the failed output, try `docker compose down`, then run `docker-compose –build` again.
+If your terminal output does not match the successful output or contains the failed output,
+run `docker compose down -v` (-v deletes cached volumes which may be interfering with pgsync), then run `docker-compose --build -d` again.
 
 4. Once running, open your browser to localhost (no port necessary).
 
 5. You will be prompted to create an account. Please follow the steps to do so, then sign in.
 
-6. Once signed in and on the home page, try searching for different things through the search bar (ex. chocolate).
+6. Once signed in and on the home page, try searching for different things through the search bar (eg., chocolate).
 
-7. Open the filters tab next to the search bar and play around with the filters. Ensure to click “Apply Filters” and then the “Search” button.
+7. Open the filters tab next to the search bar and play around with the filters. Ensure to click “Apply Filters” and
+   then the “Search” button.
 
-   - Try searching with several different permutations of filters, clicking "Apply Filters" each time before clicking "Search".
+    - Try searching with several different permutations of filters, clicking "Apply Filters" each time before clicking "
+      Search".
 
-   - While doing this, view the terminal output to see what requests are being made.
+    - While doing this, view the terminal output to see what requests are being made.
 
-  1. Try sorting functionality by clicking the “Sort By” button.
+8. Try sorting functionality by clicking the “Sort By” button.
 
 ## To run automated tests:
-Open a separate terminal and run `docker exec -it martletplace_search bash`
+
+Once the application is built using `docker-compose up --build -d`, open a separate terminal and run `docker exec -it martletplace_search bash`
 
 In the resulting bash terminal, run `pytest src/test_server.py`
```

Comment:
Could condense this into just `docker exec -it martletplace_search pytest src/test_server.py` but obviously not a big deal